xml/art_slert_hw_testing.xml

```diff
@@ -176,95 +176,95 @@
       latencies. It is caused by the behavior of underlying hardware or
       firmware, independent of Linux itself (system management
       interrupts, SMIs). </para>
 
     <para> The hardware latency detector module works by making all CPUs
       busy for a configurable amounts of time. This is done by calling
-      <systemitem class="other" otherclass="function"
-       >stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
+      <systemitem class="other" otherclass="function">stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
       for some period, and then looking for gaps in the TSC data.
       These gaps indicate times when the polling was disturbed by
       System Management Interrupts (SMI).</para>
 
     <para> To start using hardware latency detector, use as
-      &rootuser;: </para>
+      <systemitem class="username">root</systemitem>: </para>
 
-    <screen>modprobe hwlat_detector enabled=1 threshold=10</screen>
+    <screen>hwlatdetect threshold=10</screen>
 
     <para> The <literal>threshold</literal> value is the minimum latency
-      value to be considered in microseconds. Use the value that is
-      relevant for your particular situation. </para>
+      value to be considered in microseconds. Use the value which
+      is relevant for your particular situation. </para>
 
-    <para> After the module is loaded, the
-        <filename>/sys/kernel/debug/hwlat_detector/</filename> directory
-      is created. The directory contains several files: </para>
+    <para>
+        <command>hwlatdetect</command> has the below options: </para>
 
     <variablelist>
       <varlistentry>
-        <term><filename>count</filename></term>
+        <term><literal>--duration="time"{s,m,d}</literal></term>
+        <listitem>
+          <para> Run the detector logic for the specified duration. The duration is a base 10
+              integer number that defaults to a value in seconds. An optional suffix may be
+              specified to indicate minutes, hours or days. </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><literal>--threshold="microsecond"</literal></term>
         <listitem>
-          <para> Contains the number of latency spikes (larger than
-              <literal>threshold</literal>) observed since the last
-            reset. </para>
+          <para> Specify the TSC gap used to detect an SMI. Any gap value greater than "threshold" is
+              considered to be the result of an SMI occurring. </para>
         </listitem>
       </varlistentry>
       <varlistentry>
-        <term><filename>enable</filename></term>
+        <term><literal>--hardlimit="microsecond"</literal></term>
         <listitem>
-          <para> The global enable/disable toggle
-              (<literal>0/1</literal>), resets the count. </para>
+          <para> The test is considered to fail if a value above the hardlimit occurs. </para>
```

Comment:
```suggestion
          <para>
           The test fails if a value above the <literal>hardlimit</literal> occurs.
          </para>
```

Reply:
I am still not sure about the removal of "consider" here, because it imo removes relevant nuance.

```suggestion
          <para> The test is considered to have failed if a value above the hardlimit occurs. </para>
```